api/src/main/java/jakarta/config/inject/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2021 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Jakarta Config annotations to be used with a dependency injection engine, such as CDI.
+ */
+package jakarta.config.inject;
```

Comment:
I thought we don't want to have OSGI dependencies in Jakarta Config, or it's pom.

There is precisely 0 usages in all of eclipse-ee4j of that annotation in source.

https://github.com/search?q=org%3Aeclipse-ee4j+%22%40org.osgi.annotation.versioning.Version%22&type=code

Reply:
Agree

Reply:
> The PR is not functional yet. It doesn't even implement the most basic features. It's basically an empty skeleton.

Yes, it is as I said on the last meeting - this tries to get the stuff that we agree on (and is general).
There was never an intention to make this a "full" API - we need to resolve tree structure and mutability (and some other things) first.
Nevertheless it would be easier to add those as PRs to the existing skeleton, rather than doing a full PR with everything